src/main/java/com/github/manolo8/darkbot/gui/titlebar/MainTitleBar.java

```diff
@@ -31,21 +41,100 @@ public MainTitleBar(Main main, MainGui frame) {
+                add(createRightPanel(statsManager), BorderLayout.EAST);
+            }
+            title.setHorizontalAlignment(SwingConstants.CENTER);
         }
 
-    }
+        private JComponent createLeftPanel(StatsManager statsManager) {
+            JPanel panel = new JPanel(new GridLayout(2, 1));
+
+            JLabel tick = createLabel("tick_time", false, null,
+                    () -> "Tick time\n\n" + statsManager.getTickStats());
+            JLabel ping = createLabel("ping", false,
+                    color -> {
+                        float hue = 1 - Math.min(500, Math.max(1, main.statsManager.getPing())) / 500f;
+                        return Color.getHSBColor(hue * 0.35f, 1f, 0.75f);
+                    }, () -> "Game ping\n\n" + statsManager.getPingStats());
+
+            statsManager.getPingStats().setListener(ping::setText);
+            statsManager.getTickStats().setListener(tick::setText);
+
+            panel.add(tick);
+            panel.add(ping);
+
+            return panel;
+        }
+
+        private JComponent createRightPanel(StatsManager statsManager) {
+            JPanel panel = new JPanel(new GridLayout(2, 1));
+
+            JLabel cpu = createLabel("cpu", true, null, () -> "CPU usage\n\n" + statsManager.getCpuStats());
+            JLabel ram = createLabel("ram", true, null,
+                    () -> "Process RAM usage\n\n" + statsManager.getMemoryStats()
+                            + "\n\nHeap=" + (Runtime.getRuntime().totalMemory() >> 20) + "MB"
+                            + "\nIn-game=" + main.facadeManager.stats.getMemory()  + "MB");
+
+            statsManager.getCpuStats().setListener(cpu::setText);
+            statsManager.getMemoryStats().setListener(ram::setText);
 
+            panel.add(cpu);
+            panel.add(ram);
+
+            return panel;
+        }
+
+        private JLabel createLabel(String iconName, boolean alignRight,
+                                   Function<Color, Color> colorFilter, Supplier<String> tooltipSupplier) {
+            FlatSVGIcon icon = UIUtils.getSVGIcon(iconName, 10, 10);
+            if (colorFilter != null) icon.setColorFilter(new FlatSVGIcon.ColorFilter(colorFilter));
+
+            JLabel label = new JLabel(icon) {
+                @Override
+                public String getToolTipText(MouseEvent e) {
+                    return tooltipSupplier.get();
+                }
+            };
+            ToolTipManager.sharedInstance().registerComponent(label);
+            label.setFont(label.getFont().deriveFont(10f));
+
+            if (alignRight) {
+                label.setHorizontalAlignment(SwingConstants.RIGHT);
+                label.setHorizontalTextPosition(SwingConstants.LEFT);
+            } else label.setHorizontalAlignment(SwingConstants.LEFT);
+
+            Dimension preferredSize = label.getPreferredSize();
+            label.setPreferredSize(new Dimension(40, preferredSize.height));
+
+            label.setText("0");
```

Comment:
```suggestion
            label.setText("...");
```
i'd rather start out at ... than a plain 0